modules/programs/qutebrowser.nix

```diff
@@ -29,7 +29,13 @@ let
   formatKeyBindings = m: b:
     let
       formatKeyBinding = m: k: c:
-        ''config.bind("${k}", "${escape [ ''"'' ] c}", mode="${m}")'';
+        ''
+          config.bind("${k}", "${
+            if builtins.isList c then
+              escape [ ''"'' ] (concatStringsSep " ;; " c)
+            else
+              escape [ ''"'' ] c
+          }", mode="${m}")'';
```

Comment:
```suggestion
        ''config.bind("${k}", "${escape [ ''"'' ] c}", mode="${m}")'';
```

Reply:
I don't think you will need this block anymore. `c` is already coerced to a string, so `builtins.isList c` should never occur.

Reply:
You're right :) I accidentally the commit that I pushed and it was only half changed.